### PR TITLE
blog: raspberry pi and treehouses remote

### PR DESCRIPTION
**Pull request by Kid243**
just an initial thought about how to get started with treehouses.

### CODE REVIEW

**Comment by Kid243** [comment]
https://raw.githack.com/treehouses/treehouses.github.io/intro_steps/#!pages/intro/initial-steps.md

**Comment by Kid243** [comment]
https://raw.githack.com/treehouses/treehouses.github.io/intro_steps/#!pages/blog/2019-07-29-rpi-and-treehouses-remote.md

File: pages/blog/2019-07-29-rpi-and-treehouses-remote.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Initial Steps to Get Started with Treehouses
```
Comment (Liew211) [inline pages/blog/2019-07-29-rpi-and-treehouses-remote.md:1]: Could use an introduction paragraph, just maybe talking about treehouses, what it is, what it does, etc

Comment (Liew211) [inline pages/blog/2019-07-29-rpi-and-treehouses-remote.md:1]: lowercase treehouses

File: pages/blog/2019-07-29-rpi-and-treehouses-remote.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+## 1st Step
```
Comment (Liew211) [inline pages/blog/2019-07-29-rpi-and-treehouses-remote.md]: Change all steps to Step 1, Step 2, etc

File: pages/blog/2019-07-29-rpi-and-treehouses-remote.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+Assuming you have an android device and/or using an android simulator; there are two ways of going about it:
```
Comment (Liew211) [inline pages/blog/2019-07-29-rpi-and-treehouses-remote.md]: Add a double space at the end of this line to add line break

File: pages/blog/2019-07-29-rpi-and-treehouses-remote.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+Use treehouses tor to display your onion address
+
+Once you see the display of your onion address that concludes your 6th Step
+## 7th Step
+In order to verify that the previous step is working, we publish the onion address on the [treehouses gitter chat](https://gitter.im/open-learning-exchange/treehouses)
+Here are the commands to accomplish the step:
+Type treehouses tor notice to verify if tor transmit to the chat
+Type treehouses tor notice on to appear in the gitter treehouses chat
+
+N.B.: you can type treehouses tor notice off to disable your presence from the chat
```
Comment (Liew211) [inline pages/blog/2019-07-29-rpi-and-treehouses-remote.md]: Can capitalize "Android", "Raspberry Pi", maybe "Bluetooth"

File: pages/blog/2019-07-29-rpi-and-treehouses-remote.md (hunk 1 above)
Comment (Kid243) [reply pages/blog/2019-07-29-rpi-and-treehouses-remote.md:1]: Alright

**Comment by Liew211** [comment]
@Kid243 Should probably upload the images for this, unless @dogi has said otherwise?

**Pull request closed without merging**